### PR TITLE
Update relocations.md

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

**Comment by lynn915** [comment]
I see two typos that need to be addressed (an extra period, and a typo in the word "written").

File: operations/workplace/people/relocations.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
 # Relocations
 
-One of the principles of Mattermost’s Remote First culture is the ability to work from anywhere. Working from anywhere means more time with loved ones, no commute, and the ability to control everything about your workspace. Staff agreements are conditioned upon the country/region of residence noted in a staff member’s contract. Location change requests/relocations should be discussed with HR and your MLT member as there may be time zone or location requirements for specific roles and teams. Staff must receive written approval from the company prior to a country or region relocation.
+One of the principles of Mattermost’s Remote First culture is flexibility. This means more time with loved ones, no commute, complete control over your workspace, and the freedom to work from various locations..
```
Comment (cwarnermm) [inline operations/workplace/people/relocations.md]:
```suggestion
One of the principles of Mattermost’s Remote First culture is flexibility. This means more time with loved ones, no commute, complete control over your workspace, and the freedom to work from various locations.
```

File: operations/workplace/people/relocations.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
 **Relocation policy**
 
-FT staff that wish to relocate outside of their contractual region/location for a period of 6 months or longer require MLT and HR approval and follow the Relocation Request Process.
+Full-time staff wishing to relocate outside their contractual region/location for 6 months or longer must follow the Relocation Request Process:
 
-1. Team member to seek manager approval for relocation.
-2. Manager to determine whether: (i) Mattermost is open to hire in the desired location; (b) the time zone shift (if applicable) can be supported by the team/role (where applicable); and (iii) the role is needed in the desired location (where applicable).
-3. Manager to initiate relocation review by completing the [Request for Relocation Form](https://na3.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=87cb016a-463c-40ab-8e56-24a5d7918c13&env=na3&acct=533bad41-ea71-442d-b8aa-6b3d45e971d6&v=2).
-4. MLT, HR, and Total Rewards to advise on next steps, including compensation review where applicable.
+1. **Manager Approval**: Team member to seek initial writeen approval from their manager for the relocation.
```
Comment (cwarnermm) [inline operations/workplace/people/relocations.md]:
```suggestion
1. **Manager Approval**: Team member seeks initial written approval from their manager for the relocation.
```

File: operations/workplace/people/relocations.md [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
+2. **Permitted Locations**: Relocations are only permitted to Mattermost's existing entity locations and to Mattermost Talent Centers through Employers of Record (EOR).
+3. **HR and MLT Relocation Review**: Manager to forward their written approval to HR to initiate a relocation review with MLT to determine: (a) If the time zone shift (if applicable) can be supported by the team/role, (b) if the role is needed in the desired location.
```
Comment (cwarnermm) [inline operations/workplace/people/relocations.md]:
```suggestion
3. **HR and MLT Relocation Review**: Manager forwards their written approval to HR to initiate a relocation review with MLT to determine: (a) If the time zone shift (where applicable) can be supported by the team/role; (b) if the role is needed in the desired location.
```

**Review by cwarnermm — DISMISSED** [review]

**Review by natalie-hub — APPROVED** [review]
Thanks Lynn, looks good.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**